navit/speech/iphone/VSSpeechSynthesizer.m

```diff
@@ -0,0 +1,86 @@
+//
+//  VSSpeechSynthesizer.m
+//  speech_iphone
+//
+//  Created by olf on 27.05.21.
+//
+
+#import <Foundation/Foundation.h>
+#include "VSSpeechSynthesizer.h"
+#import <AVFoundation/AVFoundation.h>
+
+AVSpeechUtterance *utterance;
+AVSpeechSynthesizer *synth;
+
+@implementation VSSpeechSynthesizer
+
+- (id)init {
+    self = [super init];
+
+    if (self) {
+        utterance = [AVSpeechUtterance speechUtteranceWithString:@"Welcome to navit"];
```

Comment:
This String can't be translated, right?

Reply:
I removed that, as it was not configurable and a user might not like it. Was more for testing so I didn't need to enter navigation mode to have speech output.